Makefile.am

```diff
@@ -235,13 +268,92 @@ dist_bin_SCRIPTS += eos-updater-prepare-volume/eos-updater-prepare-volume
 
 dist_man8_MANS += eos-updater-prepare-volume/docs/eos-updater-prepare-volume.8
 
+# Flatpak installer utility library
+noinst_LTLIBRARIES += eos-updater-flatpak-installer/libeos-updater-flatpak-installer-@EUU_API_VERSION@.la
```

Comment:
I would prefer it if the library (`libeos-updater-flatpak-installer`) were in its own directory: `$top_srcdir/libeos-updater-flatpak-installer`, rather than sharing `$top_srcdir/eos-updater-flatpak-installer` with `main.c` and the service files needed for the daemon. The unit tests for the library should similarly be in `libeos-updater-flatpak-installer/tests`. See how `$top_srcdir/libeos-update-server/` and `$top_srcdir/eos-update-server/` are split. Sorry, I should have pointed out that example more obviously/explicitly earlier. I can split them up later on if you don’t have time to do it.

My thinking is that the `libeos-updater-flatpak-installer` library could be added in one commit (along with its tests), and then a subsequent commit could add `eos-updater-flatpak-installer/main.c` and all the man pages and service files needed for the daemon. That would further split up the review burden so that this could all be reviewed and committed piecewise.

If you can find time to do this, and have any questions about it, grab me on Slack. If you can’t find time, let me know and I’ll try to find time to do it.